community.md

```diff
@@ -0,0 +1,63 @@
+# Content for NodeJS Community Webpage
+
+## Introduction
+
+NodeJS is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.
+
+In its earliest times, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.
+
+Node.js foundation is an open source community, and anyone can become a part of it by contributing to its projects. Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and Node.js foundation is dedicated to making you feel welcome.
+
+## Code of Conduct
+
+Like every other community, the Node.js foundation has also defined some rules which every member needs to uphold. These rules are there to avoid any kind of misconduct and harassment.
+
+[Read the full code of conduct](https://github.com/nodejs/outreach/blob/master/CODE_OF_CONDUCT.md)
+
+## Want to Contribute
+
+Node.js is an Open Source community, and all kinds of contributions — documentation, bugs reporting, core contribution — are welcomed. Irrespective of the size of the contribution, we encourage you to be a part of this initiative and own this Open Source project. If you miss a feature, request it or maybe help to build it.
+
+- [Comm Comm Meetings](https://github.com/nodejs/community-committee)
+
+- [Core Contributions](https://github.com/nodejs/node)
+
+- [Reporting a Bug](https://github.com/nodejs/node/issues)
+
+- [Requesting a feature](https://github.com/nodejs/node/issues)
+
+
+## Node.js news
+
+### Stay in Touch
+
+Subscribe to the [NodeJS official monthly mailing list](https://us14.campaign-archive.com/home/?u=c7c2e114a827812354112c23b&id=f006b61f29) to get all the latest update related to the NodeJS.
+
+### Twitter
+
+For the exciting news about the Node.js, [follow @nodejs on twitter](https://twitter.com/nodejs).
+
+## Stuck at Somewhere
+
+### Node.js Help Repository
+
+Because of the level of activity in the `nodejs/node` is so high, we created a separate repo for helping purpose. [Check it out now](https://github.com/nodejs/help/issues). There you'll find a lot of questions or requests for general help using Node.js. If you still can't find the solution to your problem, all you need to do is to open an issue. After that, someone from the Node.js foundation will contact you on that issue.
+
+### Stack Overflow
+
+Millions of developers are using Node.js on a regular basis. We have a huge community of developers on Stack Overflow where they have answered thousands of questions related to Node.js.
```

Comment:
Again, check how it's done at https://www.gatsbyjs.org/contributing/community/

Less is more.